target/linux/ramips/mt76x8/base-files/etc/board.d/02_network

```diff
@@ -152,6 +152,10 @@ ramips_setup_macs()
 	zyxel,keenetic-extra-ii)
 		wan_mac=$(macaddr_add "$(mtd_get_mac_binary factory 0x4)" 1)
 		;;
+	zyxel,keenetic-kn-1210)
+		ucidef_add_switch "switch0" \
+			"1:lan" "2:lan" "3:lan" "0:wan" "6@eth0"
+		;;
```

Comment:
This section is for mac addresses. Move this to the proper section.

Reply:
fix